TypeCobol/Compiler/Diagnostics/MessageCode.cs

```diff
@@ -52,6 +52,7 @@ public enum MessageCode
         WrongMultilineCommentMarckupPosition = 42,
         ErrorFormalizedCommentMissplaced = 43,
         MultiFormalizedCommentIndicatorMisused = 44,
-        Info = 45
+        ControlFlowGraphDiagnostic = 45,
```

Comment:
This new message category is not used. Also I noticed the `Info` number value (46 or 45 in the previous version) does not match the id in `DiagnosticMessages.csv`. Is this intended ?

Reply:
Yes, this seem to be a strange merge effect that must be fixed.

Reply:
But be carefull this also the case in develop: 7;37;2;Info: {0};0;0;

Reply:
I've changed and tested that in #1809, you can leave it as it is in this PR, just remove the ControlFlowGraphDiagnostic member, it will be fine.